src/universal/routes.jsx

```diff
@@ -22,11 +22,17 @@ import React from 'react';
 import ModuleRoute from 'holocron-module-route';
 import { Route } from '@americanexpress/one-app-router';
 import { applicationError, clearError } from '@americanexpress/one-app-ducks';
+import { getModule } from 'holocron';
+import hasChildRoutes from './utils/hasChildRoutes';
 
 const createRoutes = (store) => {
   const rootModuleName = store.getState().getIn(['config', 'rootModuleName']);
+  const RootModule = getModule(rootModuleName);
+  const rootHasChildRoutes = hasChildRoutes(RootModule);
```

Comment:
consider calling these functions inline: 
```
  const rootHasChildRoutes = hasChildRoutes(getModule(rootModuleName));
```

this prevents us leaving the root module available to other code that doesn't need it, keeping the scope slightly cleaner.